crates/release_plz/tests/all/release_pr.rs

```diff
@@ -10,15 +11,49 @@ async fn release_plz_should_set_custom_pr_details() {
     let config = r#"
     [workspace]
     pr_name = "release: {{ package }} {{ version }}"
+    pr_body = """
+    {% for release in releases %}
+    {% if release.title %}
+    ### {{release.title}}
+    {% endif %}
+    Package: {{release.package}} {{release.previous_version}} -> {{release.next_version}}
+    {% if release.changelog %}
+    Changes:
+    {{release.changelog}}
+    {% endif %}
+    {% endfor -%}
+    """
```

Comment:
The reason why the final string isn't intended correctly is because the config is indented.

Can you remove the indentation from the entire config file? I.e. The spaces at the beginning of each line.